tests/playwright/src/specs/z-podman-machine-resources.spec.ts

```
@@ -122,6 +117,16 @@ test.afterAll(async ({ runner, page, navigationBar }) => {
   }
 });
 
+// eslint-disable-next-line no-empty-pattern
+test.afterEach(async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)